dev/ToastNotifications/ToastProgressData.cpp

```diff
@@ -6,42 +6,62 @@ namespace winrt::Microsoft::Windows::ToastNotifications::implementation
 {
     uint32_t ToastProgressData::SequenceNumber()
     {
-        throw hresult_not_implemented();
+        auto lock{ m_lock.lock_shared() };
+
+        return m_sequenceNumber;
     }
-    void ToastProgressData::SequenceNumber(uint32_t /* value */)
+    void ToastProgressData::SequenceNumber(uint32_t value)
```

Comment:
Please change the parameter names to its actual proper names, i.e. `sequenceNumber`. Please apply to all functions.